gradle.properties

```diff
@@ -1,5 +1,5 @@
 org.gradle.daemon=true
 
-selenium.version=4.8.2
+selenium.version=4.9.1
```

Comment:
I assume this would also be compatible to 4.9.0
We put a minimum compatible version here

Reply:
`loadSystemProperties` is introduced in `4.9.1`.

actually we can do a trick, it's possible to make changes in the way to keep Appium client compatible with 4.8.2, 4.8.3, 4.9.0, 4.9.1: no `@Override` should be put on `loadSystemProperties`, that's a bit hacky way, does it worth?

Reply:
no, better not to. This might open new ways to f#^%k it up.